### ISSUE TITLE
Allow run now for docker image

### ISSUE THREAD

**Issue by mcclown** [InsertDisc/pattrmm#57]
Equivalent of PMM_RUN=True for PMM

I use Ofelia for scheduling the start time of my containers. I use that for PMM on my set schedule. I would expect the docker container to stop after the run completes.

**Comment by mcclown** [comment]
There's no command that is failing specifically. It's a new feature I'm asking for, which is really just an extension of the existing functionality you provide with 

`docker run --rm -it -v "/path/to/pattrmm/preferences:/preferences" -v "/path/to/pattrmm/data:/data" -v "/path/to/pmm/config:/config" ghcr.io/insertdisc/pattrmm --run`

I'm looking for an ENV variable that I can pass to the pattrmm docker container that will make it run as soon as it starts, then shutdown after the run is complete.

This functionality is provided for PMM with they PMM_RUN=True ENV variable.

**Comment by InsertDisc** [comment]
You would like the same functionality to be available as an environmental variable as well?

**Comment by markhaines** [comment]
> You would like the same functionality to be available as an environmental variable as well?

Yes please - this would mirror a feature that PMM / Kometa has - ENV to trigger automatic run at startup.

**Comment by InsertDisc** [comment]
Just added a RUN_NOW environmental variable check. Would you mind trying this out?
RUN_NOW=True

**Comment by mcclown** [comment]
Yes I will. Thank you

**Comment by InsertDisc** [comment]
This is tested and working now.
